character and persons updated

tormodv opened the pull request:
Added object properties needed for relations between fictive characters, actors, animals and puppies.

<img width="736" alt="image" src="https://user-images.githubusercontent.com/12021922/177036530-7263b9b6-6c25-40ec-bc0e-c480588a38c4.png">

JuergenGrupp reviewed (commented):
I think all changes are correct and as intended. However, maybe it it is only a matter of style, but I fear that users might be confused by the meaning of "portrays": it is not the inverse of "isPortrayedBy" but of "isPlayedBy". Couldn't we use "plays" instead of "portrays"? Another option might be to use "starringAs" and "isStarredBy" instead of "plays" and "isPlayedBy". What do you think?

tormodv commented:
> I think all changes are correct and as intended. However, maybe it it is only a matter of style, but I fear that users might be confused by the meaning of "portrays": it is not the inverse of "isPortrayedBy" but of "isPlayedBy". Couldn't we use "plays" instead of "portrays"? Another option might be to use "starringAs" and "isStarredBy" instead of "plays" and "isPlayedBy". What do you think?

Ye this was confusing. I have changed the inverse relations and added a constraint.

JuergenGrupp reviewed (approved):
Looks consistent now.
